frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -26,3 +26,15 @@ export const createLookup = <A extends K8sResourceKind>(
   }
   return {};
 };
+
+// Ceph based provisioners
+export const cephStorageProvisioners = [
+  'ceph.rook.io/block',
+  'cephfs.csi.ceph.com',
+  'rbd.csi.ceph.com',
+];
+
+// To check if the provisioner is OCS based
+export const isCephProvisioner = (scProvisioner: string) => {
+  return cephStorageProvisioners.some((provisioner: string) => scProvisioner.includes(provisioner));
```

Comment:
Does the provisioner start with or end with these values? It would be good to be more precise here.